rules-reviewed/eap6/weblogic/weblogic-ignore-references.windup.xml

```diff
@@ -1,31 +0,0 @@
-<?xml version="1.0"?>
-<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="weblogic-ignore-references" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
-
-    <metadata>
-        <description>
-            This ruleset provides references within WebLogic classes implying the class should not be scanned by windup.
-        </description>
-        <dependencies>
-            <addon id="org.jboss.windup.rules,windup-rules-java,2.7.0.Final" />
-        </dependencies>
-        <sourceTechnology id="weblogic" />
-        <targetTechnology id="eap" versionRange="[6,8)" />
-        <tag>reviewed-2016-11-22</tag>
-    </metadata>
-    <rules>
-         <javaclass-ignore reference-prefix="weblogic.ejb.container.interfaces.WLEnterpriseBean" />
-         <javaclass-ignore reference-prefix="weblogic.ejb.container.internal" />
-         <javaclass-ignore reference-prefix="weblogic.ejb.container.persistence.spi.EloWrapper"/>
-         <javaclass-ignore reference-prefix="weblogic.ejb20.interfaces"/>
-         <javaclass-ignore reference-prefix="weblogic.ejb20.internal"/>
-         <javaclass-ignore reference-prefix="weblogic.ejbgen"/>
-         <javaclass-ignore reference-prefix="weblogic.ejb20.persistence.spi.CMPBean"/>
-         <javaclass-ignore reference-prefix="weblogic.ejb20.persistence.spi.EoWrapper"/>
-         <javaclass-ignore reference-prefix="weblogic.wsee.tools.clientgen"/>
-         <javaclass-ignore reference-prefix="weblogic.rmi.internal"/>
-         <javaclass-ignore reference-prefix="weblogic.utils.PlatformConstants"/>
-         <javaclass-ignore reference-prefix="weblogic.webservice.core.rpc"/>
```

Comment:
One additional thing to these ignores, we added them mostly that we got or know that those packages are used for automatically generated stub classes. If they are not correct please link some docs too as we marked and ignored them for this specific reason. Generated classes are waste of time for decompilation and analysis.